homeassistant/components/elmax/strings.json

```diff
@@ -15,17 +15,28 @@
           "panel_id": "Panel ID",
           "panel_pin": "PIN Code"
         }
+      },
+      "reauth_confirm": {
+        "description": "Please re-authenticate with the panel.",
+        "data": {
+          "password": "[%key:common::config_flow::data::password%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "panel_id": "Panel ID",
```

Comment:
```suggestion
```
Should be removed as it's no longer used in the step

Reply:
Done!